File: src/main/java/delivery/DeliveryStatusDemo.java
package delivery;

public class DeliveryStatusDemo {
	public static void main(String[] args) {

		final DeliveryStatus deliveryStatus = new DeliveryStatus(DeliveryStatus.Stage.PREPARING, null);

		final String trackingNumber = deliveryStatus.getTrackingInfo().getTrackingNumber();
		System.out.println("trackingNumber = " + trackingNumber);
	}
}

File: src/main/kotlin/DeliveryStatusDemo.kt

fun main(args: Array<String>) {

    val deliveryStatus = DeliveryStatus(DeliveryStatus.Stage.PREPARING, null);
    val trackingNumberUnsafe = deliveryStatus.trackingInfo.trackingNumber;

    val trackingNumberSafe = deliveryStatus.trackingInfo?.trackingNumber;
    println("$trackingNumberSafe")
}
